presto-main/src/main/java/io/prestosql/connector/system/jdbc/ColumnJdbcTable.java

```diff
@@ -372,10 +370,10 @@ static int jdbcDataType(Type type)
         if (type.equals(TIME_WITH_TIME_ZONE)) {
             return Types.TIME_WITH_TIMEZONE;
         }
-        if (type.equals(TIMESTAMP)) {
+        if (type instanceof TimestampType) {
             return Types.TIMESTAMP;
         }
-        if (type.equals(TIMESTAMP_WITH_TIME_ZONE)) {
```

Comment:
Is `TIMESTAMP_WITH_TIME_ZONE` deprecated? Are there there any other usages of this? I think we should remove these contants.

Reply:
Yes, but we can’t remove them yet due to backward compatibility concerns.